deployment/README.md

```diff
@@ -17,6 +19,31 @@ Default output format [None]:
 
 You will be prompted to enter your AWS credentials, along with a default region. These credentials will be used to authenticate calls to the AWS API when using Terraform and the AWS CLI.
 
+## Logs
+
+- LogRocket (Only client, Derek, and Bryan have access) - 1 month retention
+- Rollbar (if an error) - 180 day retention
+- AWS Cloudwatch - Django app - 1 month retention
+- AWS Cloudwatch - a lot of older logs that don't expire, and some logs from Realtor and other processes
+- AWS Builtin Logging - From ECS/RDS, using AWS defaults which is 35 days or less.
+- AWS S3 logs - New django style/Cloudfront - stored indefinitely, but not setup correctly
+- AWS S3 logs - old setup - stored indefinitely, can likely be deleted?
+
+## Personally Identifiable Information Collected
+
+This is tracked for general public usage. Azavea or customer account usage is not included.
```

Comment:
I'm not sure I understand what you mean here; it kind of sounds like we give the general public access to these stats, but that's obviously not right. Does this mean we only track general public usage without collecting specific stats about individual usage?

Reply:
Yea, that is not super clear.  I revised to:

"We are tracking this for housing searchers only and not for Azavea, BHA or other management accounts."  

Is that better?

Reply:
Yes, that's much clearer. Thanks!